### PR TITLE
LPS-133123

### PR DESCRIPTION
**Pull request by mlan0**
Related Issue: https://issues.liferay.com/browse/LPS-133123

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @mlan0, can you please check if these failures are related to your changes? Thanks

**Comment by mlan0** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -21,6 +21,10 @@
 	z-index: 1200;
 }
 
+.dropdown-menu.clay-color-dropdown-menu.show {
+	position: fixed;
+}
+
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/main.scss:27]: Good job, @mlan0! :slightly_smiling_face: 

Can you please add another commit to this pull just to sort the CSS properties alphabetically?

At this moment, the properties `dropdown-*` aren't following the proper sorting.

Thanks in advance.

Comment (mlan0) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/main.scss]: Done!

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Pull request closed without merging**